package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "migrations",
-  "version": "0.1.8",
+  "version": "0.1.7",
```

Comment:
not sure how this was 0.1.8

Reply:
Need to confirm with @vinteo, either it as deliberate which would mean the package-lock.json should be updated instead. This is the npm package which we could potentially try and get access back again.. https://www.npmjs.com/package/migrations

Reply:
I pushed it up before, it just means we haven't published it yet